components/static-code-block.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+function CodeBlock({code, language}) {
+  const languageClassName = `language-${language}`
+  return (
+    <pre className={languageClassName}>
+      <code
+        className={languageClassName}
+        dangerouslySetInnerHTML={{
+          __html: code,
+        }}
+      />
+    </pre>
+  )
+}
+
+function StaticCodeBlock({code, language, summary}) {
```

Comment:
This and the above component could be combined. I was thinking small and compos-able, but in reality I doubt the `CodeBlock` component will or would ever be used outside the `StaticCodeBlock`. It is not even exported and, even if it was, I wonder how much value it adds to be a separate thing.